ibm_catalog.json

```diff
@@ -0,0 +1,98 @@
+{
+    "products": [
+      {
+        "name": "deploy-arch-ibm-observability",
+        "label": "Deployment of IBM Cloud Observability Resources",
+        "product_kind": "solution",
+        "tags": [
+          "ibm_created",
+          "target_terraform",
+          "terraform",
+          "reference_architecture",
```

Comment:
`reference_architecture` is not a valid tag, please remove it and add `logging_monitoring`

(FYI, you can see all valid tags by running the command `ibmcloud catalog filter options --all`)

Reply:
thanks for the command. I was not knowing about this.